client/src/document/molecules/localized-content-note/index.tsx

```diff
@@ -32,6 +32,11 @@ export function LocalizedContentNote({
       linkText:
         "이 페이지는 영어로부터 커뮤니티에 의하여 번역되었습니다. MDN Web Docs에서 한국 커뮤니티에 가입하여 자세히 알아보세요.",
     },
+    "pt-BR": {
+      linkText:
+        "Esta página foi traduzida do inglês pela comunidade. Saiba mais e junte-se à comunidade MDN Web Docs.",
+      url: "/pt-BR/docs/MDN/Community/Contributing/Translated_content#locais_ativos",
```

Comment:
@mdn/yari-content-pt-br Unfortunately, this page does not seem to be translated yet. Would you be interested in translating it?